chall02/mcarrete.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
+
+import sys
+
+def stringToMorse(text):
+	morse =		{'A' : '.-', 'B' : '-...', 'C' : '-.-.', 'D': '-..', 'E' : '.',
+				'F' : '..-.', 'G' : '--.', 'H' : '....', 'I' : '..', 'J' : '.---',
+				'K' : '-.-', 'L' : '.-..', 'M' : '--', 'N' :  '-.', 'O' : '---',
+				'P' : '.--.', 'Q' : '--.-', 'R' : '.-.',  'S' : '...', 'T' : '-',
+				'U' : '..-', 'V' : '...-', 'W' : '.--', 'X' : '-..-', 'Y' : '-.--',
+				'Z' : '--..', ' ' : ' '}
+	retMorse = ""
+	count = 0
+	error = "usage: ./mcarrete.py <a-zA-Z string>"
```

Comment:
The file name might change, you should make a more dynamic message:
```suggestion
	error = "usage: %s <a-zA-Z string>" %s sys.argv[0]
```